src/py/aspen/config/config.py

```diff
@@ -187,3 +189,11 @@ def DATABASE_URI(self) -> str:
     @property
     def DATABASE_READONLY_URI(self) -> str:
         raise NotImplementedError()
+
+
+class RemoteConfig(Config):
```

Comment:
i feel like a more intuitive name might be `RemoteDBConfig`? Unless we're planning on using this config for things other that a DB down the line

Reply:
It's subclassed by all configs that use a remote db, so I guess you're right.

Reply:
Updated.